server/database.js

```diff
@@ -0,0 +1,111 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
+
+const { DB_USER, DB_PASS, DB_NAME, DB_TEST_NAME } = process.env;
+const saltRounds = 10;
+const mentionSchema = new mongoose.Schema({
+  content: {
+    type: String,
+    
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  platform: {
+    type: String,
+    required: true,
+  },
+  image: {
+    type: String,
+  },
+  date: {
+    type: String,
+    required: true,
+  },
+  popularity: {
+    type: String,
+  },
+  url: {
+    type:String,
+  }
+
+});
+const mention = mongoose.model('mention', mentionSchema);
+/**
+ * Opens a connection to the MongoDB Atlas instance used, using the
+ * individual login credentials created in the root environment file.
+ *
+ * @param {string} [environment=prod] - if 'test', then will connect to the
+ * testing database. Else it will connect to the production database.
+ * @returns {void}
+ */
+const connectDB = (environment = "prod") => {
+  let dbName = DB_NAME;
+  if (environment === "test") dbName = DB_TEST_NAME;
+  const URI = `mongodb+srv://${DB_USER}:${DB_PASS}@cluster0.sd3mb.mongodb.net/${dbName}?retryWrites=true&w=majority`;
+  const options = { useUnifiedTopology: true, useNewUrlParser: true };
+  mongoose.connect(URI, options);
+};
+ 
+function createMention(mentionBody, mentionTitle, mentionPlatform, mentionImage, mentionDate, mentionPopularity, mentionUrl){
+  const newMention = new mention({content: mentionBody, title: mentionTitle, platform: mentionPlatform, image: mentionImage,date: mentionDate, popularity:mentionPopularity, url: mentionUrl});
+  newMention.save(function (err) { if (err) return console.error(err);});
+}
+function getMention(){
+  let ans =[];
```

Comment:
I don't understand why you're pushing the result of find on to an array. 

The `find` command already returns array. Check out [this resource](https://masteringjs.io/tutorials/mongoose/find) for examples. So, if I understand your code, `ans` would end up being an array with a nested array inside, and the mentions would be inside the nested array.